### PR TITLE
CVSB-4943

### PR DESCRIPTION
**Pull request by vladddragomir**

### CODE REVIEW

File: src/pages/testing/vehicle/vehicle-history-details/vehicle-history-details.ts [hunk 1]
```diff
@@ -24,10 +25,10 @@ export class VehicleHistoryDetailsPage {
   testResultHistory: any;
   testIndex: number;
   testTypeIndex: number;
-  selectedTestResult: string;
+  selectedTestResult: TestResultModel;
```
Comment (agoncearenco) [inline src/pages/testing/vehicle/vehicle-history-details/vehicle-history-details.ts]: Are you sure this type change doesn't break unit tests?

File: src/pages/testing/vehicle/vehicle-history-details/vehicle-history-details.ts [hunk 2]
```diff
@@ -24,10 +25,10 @@ export class VehicleHistoryDetailsPage {
...
   selectedTestType: TestTypeModel;
-  testTypeResults: {};
-  defaultValues: {};
+  testTypeResults;
+  defaultValues;
```
Comment (agoncearenco) [inline src/pages/testing/vehicle/vehicle-history-details/vehicle-history-details.ts]: I would add a type for defaultValues and testTypeResults.

File: src/pages/testing/vehicle/vehicle-history-details/vehicle-history-details.ts (hunk 1 above)
Comment (vladddragomir) [reply src/pages/testing/vehicle/vehicle-history-details/vehicle-history-details.ts]: yep, i've run them. actually, it was extremely wrong being a string

File: src/pages/testing/vehicle/vehicle-history-details/vehicle-history-details.ts (hunk 2 above)
Comment (vladddragomir) [reply src/pages/testing/vehicle/vehicle-history-details/vehicle-history-details.ts]: done

**Pull request closed without merging**